test/suites/drift/suite_test.go

```diff
@@ -289,7 +292,7 @@ var _ = Describe("Drift", Label("AWS"), func() {
 				},
 			})
 		})
-		DescribeTable("provisioner static drift", func(fieldName string, provisionerOption test.ProvisionerOptions) {
+		DescribeTable("provisioner drift", func(fieldName string, provisionerOption test.ProvisionerOptions) {
```

Comment:
```suggestion
		DescribeTable("Provisioner Drift", func(fieldName string, provisionerOption test.ProvisionerOptions) {
```

We've generally been capitalizing our test names